### PR TITLE
feat(presets): favicon dark mode

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
Change (fav)icons to SVG:
- support dark mode
- support multiple sizes/types with 1 icon variation

closes: [HRZ-90539](https://spryker.atlassian.net/browse/HRZ-90539)

[HRZ-90539]: https://spryker.atlassian.net/browse/HRZ-90539?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/template/application/src/services/initializers/theme-meta.initializer.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import { inject } from '@spryker-oryx/di';
+import { featureVersion } from '@spryker-oryx/utilities';
 
 export const ThemeMetaInitializer = `${AppInitializer}ThemeMeta`;
 
 export class DefaultThemeMetaInitializer implements AppInitializer {
   constructor(protected metaService = inject(PageMetaService, null)) {}
 
   initialize(): void {
+    if (featureVersion >= '1.3') return;
```
Comment (tolerants) [inline libs/template/application/src/services/initializers/theme-meta.initializer.ts:15]: Shouldn't we deprecate it then?

File: libs/template/application/src/services/initializers/theme-meta.initializer.ts [hunk 2]
```diff
@@ -16,6 +18,15 @@ export class DefaultThemeMetaInitializer implements AppInitializer {
...
....4424 35.9396 14.7123 36.3717C14.9767 36.7942 15.2741 37.1985 15.6022 37.5816C15.6527 35.1998 16.0987 32.8389 16.9248 30.5801C17.3058 29.5412 17.7683 28.529 18.3086 27.5512C18.8172 26.6306 19.3983 25.7452 20.0471 24.9018' /><path opacity='0.7' d='M18.0837 0C18.3028 0.760402 18.4789 1.53067 18.6112 2.30772C18.7519 3.1143 18.8574 3.95168 18.9369 4.81988C19.0164 5.68807 19.0745 6.60948 19.1143 7.55329C19.154 8.4971 19.1785 9.50253 19.1968 10.5262C19.2397 12.9067 19.0439 14.8237 18.703 16.4229C18.4159 17.8177 17.9429 19.1749 17.2947 20.4642C16.7717 21.4524 16.1924 22.4148 15.5593 23.3474C14.9966 24.224 14.5027 25.0474 14.1954 26.008C14.1847 26.0332 14.174 26.057 14.1633 26.0808C14.1526 26.1046 14.1404 26.1298 14.1296 26.1537L14.0975 26.2265C14.0868 26.2517 14.0761 26.2755 14.0654 26.3007L20.2305 23.5869C20.5454 21.7777 20.7488 20.0861 20.8604 18.5009C20.9705 16.9606 20.998 15.5239 20.969 14.1866C20.9384 12.8787 20.8528 11.666 20.732 10.5458C20.6112 9.42551 20.4613 8.43269 20.2947 7.51688C20.1693 6.82373 20.0256 6.14598 19.8681 5.48363C19.7106 4.82128 19.5378 4.18414 19.3513 3.5568C19.1647 2.92946 18.966 2.32732 18.7534 1.73639C18.5409 1.14546 18.3192 0.560125 18.0837 0Z' /></g></svg>",
```
Comment (tolerants) [inline libs/template/application/src/services/initializers/theme-meta.initializer.ts]: Hm... it looks strange we have size 32x32 and a viewBox='0 0 30 42'. Can you double check?

File: libs/template/presets/src/meta.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      href: 'https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap',
+      media: 'all',
+    },
+  },
+];
+
+export const bodyStyles = (): ElementDefinition[] => [
+  {
+    name: 'style',
+    attrs: { text: ':root{font-size: 16px}body {margin: 0;}' },
```
Comment (tolerants) [inline libs/template/presets/src/meta.ts]: For storefront default font-size is 14px, isn't it?

File: libs/template/presets/src/meta.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
....4424 35.9396 14.7123 36.3717C14.9767 36.7942 15.2741 37.1985 15.6022 37.5816C15.6527 35.1998 16.0987 32.8389 16.9248 30.5801C17.3058 29.5412 17.7683 28.529 18.3086 27.5512C18.8172 26.6306 19.3983 25.7452 20.0471 24.9018' /><path opacity='0.7' d='M18.0837 0C18.3028 0.760402 18.4789 1.53067 18.6112 2.30772C18.7519 3.1143 18.8574 3.95168 18.9369 4.81988C19.0164 5.68807 19.0745 6.60948 19.1143 7.55329C19.154 8.4971 19.1785 9.50253 19.1968 10.5262C19.2397 12.9067 19.0439 14.8237 18.703 16.4229C18.4159 17.8177 17.9429 19.1749 17.2947 20.4642C16.7717 21.4524 16.1924 22.4148 15.5593 23.3474C14.9966 24.224 14.5027 25.0474 14.1954 26.008C14.1847 26.0332 14.174 26.057 14.1633 26.0808C14.1526 26.1046 14.1404 26.1298 14.1296 26.1537L14.0975 26.2265C14.0868 26.2517 14.0761 26.2755 14.0654 26.3007L20.2305 23.5869C20.5454 21.7777 20.7488 20.0861 20.8604 18.5009C20.9705 16.9606 20.998 15.5239 20.969 14.1866C20.9384 12.8787 20.8528 11.666 20.732 10.5458C20.6112 9.42551 20.4613 8.43269 20.2947 7.51688C20.1693 6.82373 20.0256 6.14598 19.8681 5.48363C19.7106 4.82128 19.5378 4.18414 19.3513 3.5568C19.1647 2.92946 18.966 2.32732 18.7534 1.73639C18.5409 1.14546 18.3192 0.560125 18.0837 0Z' /></g></svg>`;
```
Comment (tolerants) [inline libs/template/presets/src/meta.ts]: The same question about viewBox='0 0 30 42'.

**Review by tolerants — CHANGES_REQUESTED** [review]

File: libs/template/presets/src/meta.ts (hunk 3 above)
Comment (tobi-or-not-tobi) [reply libs/template/presets/src/meta.ts]: totally!

File: libs/template/application/src/services/initializers/theme-meta.initializer.ts (hunk 2 above)
Comment (tobi-or-not-tobi) [reply libs/template/application/src/services/initializers/theme-meta.initializer.ts:26]: The SVG is based on 42px width, see `logo.ts` for the source, but I made it a proper square this time around.

Comment (tobi-or-not-tobi) [reply libs/template/application/src/services/initializers/theme-meta.initializer.ts]: This icon should not be here, breaking change. We moved it to meta.ts

File: libs/template/presets/src/meta.ts (hunk 4 above)
Comment (tobi-or-not-tobi) [reply libs/template/presets/src/meta.ts]: The SVG is based on 42px width, see `logo.ts` for the source, but I made it a proper square this time around.

**Review by tolerants — APPROVED** [review]

**Pull request merged**